packages/lit-dev-content/samples/docs/controllers/names/names-controller.ts

```diff
@@ -16,13 +16,13 @@ export class NamesController {
         if (!kind?.trim()) {
           return initialState;
         }
-        const response = await fetch(`${Names.baseUrl}${Names.kindIdMap[kind]}`);
-        const result = await response.json();
-        const error = (result as Names.Error).error;
-        if (error !== undefined) {
-          throw new Error(error);
+        try {
+          const response = await fetch(`${Names.baseUrl}${kind}`);
+          const data = await response.json();
+          return data.results as Names.Result;
+        } catch(error) {
```

Comment:
Nit: `error` is not referenced. Replace with `} catch {`

Reply:
Ah, yes good point. I considered doing a `console.error(error);` before re-throwing a new error so we don't lose that info but I wasn't sure if it was necessary for this example. What do you think?

Reply:
I think it's fine removing since the error flows into the render. And not sure we expect or want to encourage samples logging into the console.